src/modules/removeTask.js

```diff
@@ -0,0 +1,10 @@
+import showTasks from './showTasks.js';
+
+const removeTask = (arr, index) => {
+  arr.splice(index, 1);
+  for (let i = index; i < arr.length; i += 1) {
+    arr[i].index = i;
+  }
+  showTasks(arr);
+};
```

Comment:
- [ ] Good job so far I can remove a task from the list but when I remove a task something happens with the index, the index is supposed to be updated correctly for example if I have 5 elements to my list and I remove the 3rds element to the list then the 4th element become the 3rds and the 5th element become the 4th. In your case, the 4th element becomes the 2nd and now we have two index identic and the 5th element becomes the 3rds instead of the 4th element according to the array. Please kindly fix that 🙏🏾 

<img width="474" alt="Screenshot 2023-05-05 at 19 32 31" src="https://user-images.githubusercontent.com/74501523/236518997-dc2aef20-8e5f-4d07-8300-0e754700c3a1.png">

<img width="474" alt="Screenshot 2023-05-05 at 19 32 54" src="https://user-images.githubusercontent.com/74501523/236519036-4a89b450-ffc8-4e7c-be09-027c70685033.png">

Reply:
Okay @cynthiainga , I will try to solve it, if you have any suggestions to make it work correctly, I'd like to hear about it, please.

Reply:
I solved it eventually :)